css/survey-task.styl

```diff
@@ -36,27 +36,28 @@ $survey-task-pill-button
 
   .modal-form-trigger
     @extends $reset-button
-    border-radius: 25px
+    background-color: #404040
     color: white
     cursor: pointer
     height: 25px
+    margin-right: 1px
     margin-bottom: 10px
     min-width: 60px
     text-transform: capitalize
 
-    &:hover
-      background-color: rgba(164, 164, 164, 0.5)
+    &:hover, &:focus
+      background-color: #666666
 
     &[aria-expanded="true"]
-      background-color: #a4a4a4
+      background-color:#666666
 
 .survey-task-chooser-characteristic
   display: block
 
 .survey-task-chooser-characteristic-menu
   .modal-form
     background-color: #dddddd
-    border-radius: 6px
+    border-radius: 0px
```

Comment:
Minor point but the syntax is `border-radius: 0` when the value of a property is zero.